tests/errors/semantic/non_blocking/PRIMITIVE_IMMUTABLE_RESTRICTION_IS.py

```diff
@@ -0,0 +1,3 @@
+@types('int', 'int')
+def compare_is_int_int(a, b):
+    return a is b
```

Comment:
We should probably add tests for the other types too

Reply:
I agree.